Makefile

```
@@ -32,12 +32,22 @@ jadx:
 	@echo "Building Graffiti for Jadx"
 	cp backends/jadx/graffiti.jadx.kts out/graffiti_v$(VERSION).jadx.kts
 
-jeb:
+jeb_packed_%:
+	@echo Packing $*
+	mkdir -p backends/jeb/packed && \
+	( (cat backends/jeb/$*.py | awk '/^#/ {print} !/^#/ {exit}') &&\
+	   echo && echo &&\
+	   python -m pybunch -r backends/jeb -e $* -so ) | cat > backends/jeb/packed/$*.py
```

```suggestion
	   python3 -m pybunch -r backends/jeb -e $* -so ) | cat > backends/jeb/packed/$*.py
```
python doesn't exist on normal machines